### DISCUSSION TITLE
How to change the Background color of the UserControl

### DISCUSSION THREAD

**Discussion by ConanNobita** [HandyOrg/HandyControl#1057] [Q&A] [answered]
How to change the Background color of the UserControl instead of default color in the library, because we want to customize the appearance of the view.

**Comment by Yopler** [comment, marked as answer]
Hi 🖐,

HandyControl doesn't change Background color for UserControl. 

If you want to customize the color of your application you can overwrite the colors provide by HandyControl. You can see here the colors : [HandyControl/src/Shared/HandyControl_Shared/Themes/Basic/Colors/](https://github.com/HandyOrg/HandyControl/tree/master/src/Shared/HandyControl_Shared/Themes/Basic/Colors)

You can overwrite them in your App.xaml.

If you want to change the Background colors, there is 3 main background colors in HandyContol : 
- RegionBrush
- SecondaryRegionBrush
- ThirdlyRegionBrush

Hope it will help ! 😄
